generator/dotnet/template/appsettings.Development.json

```diff
@@ -0,0 +1,9 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft.AspNetCore": "Warning"
+    }
+  },
+  "AllowedHosts": "*"

```

Comment:
Can this be made more secure? Allowing only particular hosts which interact with the app

Reply:
restricted to localhost